src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -1463,6 +1488,15 @@
       "BaselineVersion": "6.0.0",
       "InboxOn": {}
     },
+    "runtime.win-x64.Microsoft.NETCore.ILAsm": {
```

Comment:
I suppose the way you generated these changes was by pointing to a local folder of your build? If so I guess that would explain the additions but as Eric said we should probably remove those entries.